docs/getting-started/faq.md

```diff
@@ -1,71 +1,73 @@
 # FAQ
 
-## Why a Messenger?
+## Basics
 
-When Carl & Jarrad looked at how to achieve mass adoption for a client. We looked at things such as where the people are, how they behave on their devices, time spent on pc vs mobile - as of 2014 more time is spent on mobile than desktop, and of that time - a THIRD of it is spent within Instant Messengers.
+### What is Status?
 
-Instant Messengers have the highest retention rates, that is when you purchase an install, they become sticky and typically users won't immediately uninstall. 
+„Status is an open source messaging platform and mobile interface to interact with decentralized applications that run on the Ethereum Network.” (quoted from [Whitepaper](https://status.im/whitepaper.pdf))
 
-The average user lifetime sky rockets once their friends are involved. When you start thinking about this as a user base and how to connect that with Ethereum, things like payments and applications built ontop of Ethereum integrated into a converstional interface starts becoming like a clear path forward.
+### What does Status want to solve?
 
-Ultimately we're building a hybrid browser and messenger for us to have the best chance to cast the widest net and focus on user acquisition, staying agnostic and as close as possible to the principles Ethereum embodies.
+Status wants to solve the problem that we (the users) have different identities all over the internet. It criticized that we have spreaded to many personal information all over various platforms. Status wants to create a digital hub where we can use various services based on the Ethereum Network.
```

Comment:
"we have spreaded to many personal information all over various platforms" should be:

"we have spread too much personal information across various platforms". 

I really appreciate this PR, but cannot merge it with grammar and spelling errors like this, sorry.